docs/download.md

```diff
@@ -10,21 +11,22 @@ Use the links below to download the Apache APISIX™ from one of our mirrors.
-| 2.2     | 08/01/2020   | [source](https://www.apache.org/dyn/closer.cgi/apisix/2.2/apache-apisix-2.2-src.tgz) ([asc](https://downloads.apache.org/apisix/2.2/apache-apisix-2.2-src.tgz.asc) [sha512](https://downloads.apache.org/apisix/2.2/apache-apisix-2.2-src.tgz.sha512))
+| Version | Release Date | Downloads                                                                                                                                                                                                                                              |
+| ------- | ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| 2.2     | 08/01/2020   | [source](https://www.apache.org/dyn/closer.cgi/apisix/2.2/apache-apisix-2.2-src.tgz) ([asc](https://downloads.apache.org/apisix/2.2/apache-apisix-2.2-src.tgz.asc) [sha512](https://downloads.apache.org/apisix/2.2/apache-apisix-2.2-src.tgz.sha512)) |
 
 ## APISIX™ Dashboard
 
-| Version | Release Date | Downloads                                                                                                                                                                                                                                                                                                       |
-| ------- | ------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Version | Release Date | Downloads                                                                                                                                                                                                                                                                                                                               |
+| ------- | ------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 2.3     | 11/01/2020   | [source](https://www.apache.org/dyn/closer.cgi/apisix/dashboard/2.3/apisix-dashboard-2.3-src.tgz) ([asc](https://www.apache.org/dyn/closer.cgi/apisix/dashboard/2.3/apisix-dashboard-2.3-src.tgz.asc) [sha512](https://www.apache.org/dyn/closer.cgi/apisix/dashboard/2.3/apisix-dashboard-2.3-src.tgz.sha512))                         |
```

Comment:
@juzhiyuan 

2020->2021

Reply:
oh no

Reply:
Just updated the built files, let me update this doc.